lib/mayaUsd/render/vp2RenderDelegate/material.cpp

```diff
@@ -2780,8 +2793,7 @@ TfToken _RequiresColorManagement(
     }
 
     // Look for explicit color spaces first:
-    std::string       sourceColorSpace;
-    static const auto _mxFindColorSpace = [&sourceColorSpace](const auto& n) {
+    static const auto _mxFindColorSpace = [](const auto& n, auto& sourceColorSpace) {
```

Comment:
The lambda capture by reference was causing memory corruption. Passing the color space as a function parameter is safer in this case.